packages/zeronetworks/changelog.yml

```
@@ -1,6 +1,9 @@
 # newer versions go on top
 - version: "1.12.0"
   changes:
+    - description: Add dasboard
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/8934
```

This needs to be a new version.
```suggestion
- version: "1.13.0"
  changes:
    - description: Add dashboard.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/8934
- version: "1.12.0"
  changes:
```